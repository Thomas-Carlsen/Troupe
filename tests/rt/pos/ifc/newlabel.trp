let val l = newlabel () 
    val x = 42 raisedTo l 
in x 
end
