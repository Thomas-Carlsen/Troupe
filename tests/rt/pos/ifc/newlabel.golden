2019-04-15T12:49:14.146Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: 42@{e184cdd4-088d-4e6b-937e-41a73f97bd9b}%{}
